intro-to-arm/run

```diff
@@ -956,6 +957,96 @@ class EmbryoARMJumps(EmbryoARMBase):
             and self.val == self.emu.reg_read(UC_ARM64_REG_X1)
         )
 
+class EmbryoARMAvg(EmbryoARMBase):
+    def __init__(self, asm=None):
+        super().__init__(asm)
+        self.val_stk = [random.randint(1000000, 1000000000) for _ in range(8)]
+        self.code_load_addr = self.BASE_ADDR + self.CODE_OFFSET
+
+    def print_level_text(self):
+        print(
+            "Function calls in aarch64 are done with the branch and link instruction bl.\n"
+            "The functions return value is stored in register x0.\n\n"
+
+            "The bl instruction:\n"
+            "\tt- does a PC relative jump the specified location\n"
+            "\t- and stores the return address in the link register lr (aka x30)\n\n"
+            
+            "It is the caller's responsibility to store the existing lr value frame pointer\n"
+            "and any needed values in x0 - x15.\n\n"
+
+            "Registers x16 - x18 will be discussed later.\n"
+            "Registers x19 - x28 are caller saved.\n"
+            "The saved return address is stored in a special link register lr (aka x30).\n"
+            "The saved frame pointer is stored in a special frame register fr (aka x29).\n\n"
+
+            "Given the role of x29 and x30, it is common to see a function prologue similar to:\n"
+            "\tstp x29, x30, [sp, #-48]!\n"
+            "\tmov x29, sp\n\n"
+
+            "Here, the stack pointer is decremented to create a function frame and lr and fr are\n"
+            "stored on the stack.  The last instruction shown sets the frame pointer.  Note that\n"
+            "the stack pointer and frame pointer are equal in this case.  local variables are stored\n"
+            "ABOVE the frame pointer.  The stack pointer may decrement further when passing arguments\n"
+            "via the stack or for dynamic stack allocations (alloca).\n\n"
+
+            "Similarly, a function epilogue consists of:\n"
+            "\tldp x29, x30, [sp], 48\n"
```

Comment:
#48

Reply:
that's not even mine, that's one of yours.

Reply:
I realized that right after I commented.  I'm critiquing my own code via copy pasta.  I'll merge it in and fix up a few things in all places that were my doing.

Thanks or the PR!